hdr_html_canvas_element.md

```diff
@@ -200,12 +200,12 @@ Add a new CanvasColorMetadata dictionary:
 
 ```idl
 dictionary CanvasColorMetadata {
-  CanvasMasteringDisplayMetadata masteringDisplayMetadata;
+  CanvasColorVolumeMetadata colorVolumeMetadata;
```

Comment:
>  If the data actually held within is still mastering display information, it might imply something unintended

The data held is not mastering display information.

The data is intended to describe the contents of the image.